Firmware/MotorControl/low_level.c

```diff
@@ -789,6 +792,14 @@ bool calib_enc_offset(Motor_t* motor, float voltage_magnitude) {
         }
         encvaluesum += (int16_t)motor->encoder.encoder_timer->Instance->CNT;
     }
+
+    float estEnc = scan_range / elec_rad_per_enc;
+    float adjustedCount = fabsf((int16_t)motor->encoder.encoder_timer->Instance->CNT)-init_enc_val;
```

Comment:
I think the closing bracket of fabsf should be at the end, to include the subtraction?

Reply:
Also I would maybe change the variables to be a bit more descriptive.
estEnc -> expected_enc
adjustedCount -> abs_delta_enc